modules/installing-ocp-agent.adoc

```diff
@@ -10,16 +10,26 @@ The following procedure deploys a single-node {product-title} in a disconnected
 
 .Procedure
 
-. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest[OpenShift mirror site] and download the latest version of the installer and client tarballs that match your operating system.
+. Log in to the {product-title} web console using your login credentials.
 
-. Extract the binary by running the following command:
+. Navigate to link: https://console.redhat.com/openshift/create/datacenter.
```

Comment:
This link isn't rendering properly. 

![image](https://user-images.githubusercontent.com/7443887/220389320-076c6da4-dd06-43e9-bde4-63d103e4f251.png)

There's a space after the colon, and there's no link text

Reply:
Added link text and removed space